src/components/KPrompt/KPrompt.vue

```diff
@@ -42,6 +42,7 @@
         </p>
         <KInput
           v-model="confirmationInput"
+          :aria-label="`${confirmationPromptText[0]}&quot;${confirmationText}&quot;${(confirmationPromptText[1] ? confirmationPromptText[1] : '')}`"
```

Comment:
I don't believe you should be using encoded characters in the label. Just use a single quote.

Reply:
Done